intelmq/lib/upgrades.py

```diff
@@ -662,6 +663,23 @@ def v300_pipeline_file_removal(defaults, runtime, harmonization, dry_run):
     return messages if messages else changed, defaults, runtime, harmonization
 
 
+def v301_deprecations(defaults, runtime, harmonization, dry_run):
+    """
+    Deprecate malwaredomains parser
```

Comment:
Could you please extend that to the collector as well? There are already some examples in this file which you can copy and adapt.

And we can test this function, see `intelmq/tests/lib/test_upgrades.py`.